src/screens/Profile/components/user_info/user_info.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { Text, View } from 'react-native';
```

Comment:
Components file structure inside screens must be the same as in root components folder - create index.ts file and re-export this component, do the same in profile/components folder. Also rename Profile folder to lowercase, user_info to user-info